File: src/com/tbilisi/bus/data/BusStop.java
package com.tbilisi.bus.data;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;
import io.realm.annotations.RealmClass;

@RealmClass
public class BusStop extends RealmObject {
    @PrimaryKey
    private int id;
    private String name;
    private double lat;
    private double lon;

    public BusStop() {}

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setLat(double lat) {
        this.lat = lat;
    }

    public void setLon(double lon) {
        this.lon = lon;
    }

    public double getLat() {
        return this.lat;
    }

    public double getLon() {
        return this.lon;
    }
}

File: src/com/tbilisi/bus/data/BusStop.kt
package com.tbilisi.bus.data

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import io.realm.annotations.RealmClass

@RealmClass
public open class BusStop : RealmObject() {
    @PrimaryKey
    public open var id: Int = 0
    public open var name: String = ""
    public open var lat: Double = 0.toDouble()
    public open var lon: Double = 0.toDouble()
}
